app/src/main/java/net/bible/service/format/osistohtml/osishandlers/OsisToBibleSpeak.kt

```diff
@@ -152,6 +152,8 @@ class OsisToBibleSpeak(val speakSettings: SpeakSettings, val language: String) :
         s = s.replace("`", "'")
         s = s.replace("´", "'")
         s = s.replace("’", "'")
+        s = s.replace("(", "")
+        s = s.replace(")", "")
```

Comment:
Should we replace it with space instead? "Test(test two)" would render as "Test test two". Though then "Something (test). Something" would render as "Something test . Something" (period would render alone). 

Should we just replace sad faces (":(" -> ":")?

Reply:
Replacing with space would probably work. I tried adding space before and after parenthesis but that did not work, I'm guessing later step may remove space again.

I've also heard TTS say wink-face and smile-face 😄 so we'd have to replace those too if we wanted to go that route...

Perhaps add a . (dot) between :) ;) :( ;( so they would become :.) ;.) etc...
-- however, I find that doesn't work at this level because I in Acts 8:15-16 the colon is separate (alone) from the parenthesis...